src/NHibernate/Cfg/MappingSchema/HbmOneToOne.cs

```diff
@@ -18,9 +18,13 @@ public string Access
 			get { return access; }
 		}
 
+		// 6.0 Todo : remove XmlIgnore after removing the setter. See #3607 fix.
```

Comment:
Do we actually care if it stays here or removed?

Reply:
For consistency with other mapping classes like `HbmManyToOne`, I would rather remove it.